docs/about/getting-started.md

```diff
@@ -18,7 +18,10 @@ The application receives a number of command-line flags including the requests t
 | -grpc-headers                     | strings | N/A                         | gRPC headers to be sent with warm up requests. To send multiple headers define this flag for each header                                                                           |
 | -grpc-requests                    | strings | N/A                         | gRPC requests to be sent. Request is in '\<service\>\<method\>\[:message\]' format. E.g. health/ping:{"key": "value"}. To send multiple requests define this flag for each request |
 | -http-headers                     | strings | N/A                         | Http headers to be sent with warm up requests. To send multiple headers define this flag for each header                                                                           |
-| -http-requests                    | string  | N/A                         | Http request to be sent. Request is in '\<http-method\>:\<path\>\[:body\]' format. E.g. post:/ping:{"key": "value"}. To send multiple requests define this flag for each request   |
+| -http-requests                    | string  | N/A                         | Http request to be sent. Request is in `<http-method>:<path>[:body]` format. E.g. `post:/ping:{"key": "value"}`. To send multiple requests define this flag for each request       |
+| -fail-readiness-enabled           | bool    | false                       | If set to true readiness will fail if there were issues with the warmup                                                                                                            |
+| -fail-readiness-no-requests       | bool    | true                        | Readiness will fail if no requests were sent. Requires `fail-readiness-enabled` to be set to true                                                                                  |
```

Comment:
Unless you know the internals of Mittens the description is a bit cryptic. I think a better wording would be "If true, the readiness check will fail when Mittens is unable to run the warmup for the target application.